package.json

```diff
@@ -58,6 +58,7 @@
     "prompts": "^2.3.0",
     "rxjs": "^6.5.4",
     "simple-git": "^2.0.0",
+    "slash": "^3.0.0",
```

Comment:
Basically `.replace(/\\/g,...` but more readable (signals intent, DRY) and handles a few esoteric edge cases where invalid Windows paths could be produced.